website/docs/cdktf/examples-and-guides/refactoring.mdx

```diff
@@ -0,0 +1,255 @@
+---
+page_title: Refactoring - CDK for Terraform
+description: >-
+  As your CDKTF application grows in size and complexity, the need to move resources between stacks becomes more common. This guide will walk you through the process of refactoring your stacks.
+---
+
+# Refactoring
+
+As your CDKTF application grows in size and complexity, the need to move resources between stacks becomes more common. This guide will walk you through the process of refactoring your stacks. In general you can change the names of stacks freely without an effect on the synthesized code. If you change the name of a resource this might lead to a re-creation of the resource, especially if you move the resource from one stack to another.
+
+## Moving resources inside a stack
```

Comment:
@DanielMSchmidt all of these should be heading case (all major words capitalized). That's what the docs style guide says. I won't go through and make comments on each one, but would you be willing to go through and do that for all the titles?

Thank you :) 
```suggestion
## Moving Resources Inside a Stack
```